src/main/java/org/everit/i18n/propsxlsconverter/PropsXlsConverter.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2011 Everit Kft. (http://www.everit.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.everit.i18n.propsxlsconverter;
+
+import org.apache.commons.cli.BasicParser;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.everit.i18n.propsxlsconverter.api.ConverterFunctionService;
+import org.everit.i18n.propsxlsconverter.internal.ConverterFunctionServiceImpl;
+
+/**
+ * The main class to manage export or import function.
+ */
+public final class PropsXlsConverter {
+
+  private static final String ARG_FILE_REGULAR_EXPRESSION = "fileRegularExpression";
+
+  private static final String ARG_FUNCTION = "function";
+
+  private static final String ARG_LANGUAGES = "languages";
+
+  private static final String ARG_WORKING_DIRECTORY = "workingDirectory";
+
+  private static final String ARG_XLS_FILE_NAME = "xlsFileName";
+
+  private static Options createOptions() {
+    Options options = new Options();
+    options.addOption("f", ARG_FUNCTION, true, "The function (import or export). Mandatory.");
+    options.addOption("xls", ARG_XLS_FILE_NAME, true,
+        "To import or export file names (vivelab-web.xls). Mandatory.");
```

Comment:
The excel file used for the import or export function.\nFor example: translation.xls.\nMandatory.